apollo-sample/src/main/java/com/example/apollographql/sample/MainActivity.java

```diff
@@ -18,25 +21,31 @@
 public class MainActivity extends AppCompatActivity {
   private static final String TAG = "MainActivity";
 
+  private SampleApplication application;
+
   @Override protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
     final TextView txtResponse = (TextView) findViewById(R.id.txt_response);
-    SampleApplication application = (SampleApplication) getApplication();
-    application.githuntApiService()
-        .githuntFeed(new FeedQuery(FeedQuery.Variables.builder()
-                .limit(10)
-                .type(FeedType.HOT)
-                .build()))
+    application = (SampleApplication) getApplication();
+
+    Observable<Response<FeedQuery.Data>> query = Observable.fromCallable(new Callable<Response<FeedQuery.Data>>() {
```

Comment:
Wow this looks really ugly.  No fault of yours. Let's try this for now (until I get the Rxjava Call Adapter stuff in).  Move `executeFeedQuery` to `SampleApplication` have that method take a param of limit and return an observable of the Response.  This way the activity can just do application.executeFeedQuery(10).subscribe()